src/syntax/types.rkt

```diff
@@ -20,8 +20,10 @@
 (define-type bool (boolean? boolean?)
   identity identity)
 
-(define (value-of type) (type-exact? (hash-ref type-dict type)))
-(define (bigvalue-of type) (type-inexact? (hash-ref type-dict type)))
+(define (value-of type) (type-exact? (hash-ref type-dict type))) ; used once (eval-application in src/programs.rkt)
+(define (bigvalue-of type) (type-inexact? (hash-ref type-dict type))) ; not used
 
-(define (value? x) (for/or ([(type rec) (in-hash type-dict)]) ((type-exact? rec) x)))
+; used 3 times (src/float.rkt and src/syntax/syntax.rkt)
+; counterpart value? is in src/interface.rkt since the definition of what is a value is
+; now repr specific
 (define (bigvalue? x) (for/or ([(type rec) (in-hash type-dict)]) ((type-inexact? rec) x)))
```

Comment:
I think we should eventually retire all of these or something, but I guess for now we can keep them. Let's get this merged, there are always more chances to clean house.